test/tests/neml_stochasticSimple/tests

```diff
@@ -0,0 +1,16 @@
+[Tests]
+  design = 'neml.md'
+  issues='#78'
+  [neml_simple_stochastic]
+    type = 'CSVDiff'
+    input = 'stochastic_driver.i'
+    csvdiff = 'out_avg_stress_zz_collected.csv'
+    required_objects = NEMLStressVariableInput
+    design = 'ComputeNEMLStressInput.md'
+    requirement = 'BlackBear shall be capable of using the NEML library to compute the
+                   response of a linearly elastic material parsed from xml.
+                   Parameters defined in the xml can be overwritten by up to five
+                   variables in the input file.  The BlackBear simulations and NEML variabls 
```

Comment:
This isn't really written like a requirement. It should be something like:
`BlackBear shall provide the ability to use the NEML library to model material response, and allow for parameters defined in the xml file to be overridden in the BlackBear input file`

Reply:
I used your requirement